src/actions/leaderboardActions.js

```diff
@@ -0,0 +1,48 @@
+import { FETCH_LEADERBOARD } from './types';
+
+export const fetchLeaderboard = () => async dispatch => {
```

Comment:
airbnb style guide [prefers default export](https://github.com/airbnb/javascript#modules--prefer-default-export)

Reply:
I don't actually like this exactly, I believe google prefers no default exports; it just depends how much we're going to follow the rules